src/devices_models/devices/AC_branches.jl

```diff
@@ -543,6 +586,14 @@ function add_constraints!(
     U <: AbstractBranchFormulation,
     V <: PM.AbstractActivePowerModel,
 }
+    error("slacks here")
```

Comment:
debug?

Reply:
This is a deeper bug, this code isn't being called.

Reply:
after testing this is stale code the whole function